exporter/kafkaexporter/config.go

```diff
@@ -44,6 +44,9 @@ type Config struct {
 
 	// Authentication defines used authentication mechanism.
 	Authentication Authentication `mapstructure:"auth"`
+
+	// MessageMaxSize defines the max size of the message can be sent by sarama.
+	MessageMaxSize int `mapstructure:"message_max_size"`
```

Comment:
What is the unit? Probably better to avoid "int" and use one fixed size like int32 or int64

Reply:
It is byte in sarama's config. Maybe int64 is better.

Reply:
@bogdandrutu I changed it to int64. But it seemed a little weird. Sarama's config is int so I need changed it to int again when I set it to sarama's config.

Reply:
int is fine then :). Please comment at least about the size

Reply:
oh yes, thanks! Btw, I have changed the config `MessageMaxSize` to `MessageMaxSizeMb`. In our actual use scenarios, I find this more convenient.:)